icons/stylus_extension.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Stylus</title><path d="M24 10.688V13.5h-1.5v7.688c0 1.546-1.266 2.812-2.813 2.812H4.312C2.766 24 1.5 22.734 1.5 21.187V13.5H0v-2.813l1.5.001V2.813C1.5 1.266 2.766 0 4.312 0h15.375C21.234 0 22.5 1.266 22.5 2.813v7.875H24ZM4.969.938c-1.313 0-2.344 1.031-2.344 2.343v7.407s-.094.843-1.5.843v1.125c1.406 0 1.5.844 1.5.844v7.219c0 1.312 1.031 2.344 2.344 2.344h14.062c1.313 0 2.344-1.032 2.344-2.344V13.5s.094-.844 1.5-.844v-1.125c-1.406 0-1.5-.843-1.5-.843V3.281c0-1.312-1.031-2.343-2.344-2.343H4.969ZM20.375 13.5v7.219c0 .752-.591 1.344-1.344 1.344H4.969c-.753 0-1.344-.592-1.344-1.344v-7.147c.004-.061.001-.143-.013-.231-.012-.082-.077-.52-.483-.949a2.1008 2.1008 0 0 0-.348-.298c.142-.097.256-.2.348-.298.406-.429.471-.868.483-.949a.2912.2912 0 0 0 .006-.045.8768.8768 0 0 0 .007-.115V3.281c0-.752.591-1.343 1.344-1.343h14.062c.753 0 1.344.591 1.344 1.343v7.407l.007.114s.002.018.006.045c.012.081.077.52.483.949.092.098.206.201.348.298-.142.097-.256.2-.348.298-.406.429-.471.867-.483.949-.004.026-.006.044-.006.044s-.007.667-.007.115ZM12.844 2.953c-4.125 0-6.656 2.297-6.656 5.297 0 2.531 1.921 4.125 4.828 5.203 2.109.703 2.953 1.313 2.953 2.438 0 1.125-.938 1.875-2.766 1.875-1.687 0-3.375-.516-4.406-1.078L6 19.922c.984.562 2.953 1.125 4.969 1.125 4.781 0 7.031-2.531 7.031-5.438 0-2.484-1.406-4.125-4.5-5.25-2.297-.843-3.281-1.359-3.281-2.484 0-.891.89-1.687 2.578-1.687 1.687 0 2.953.515 3.609.843l.891-3.187c-1.031-.469-2.484-.891-4.453-.891Z"/></svg>
```

Comment:
Looking at the source, I think I would opt for the following treatment 
![image](https://github.com/simple-icons/simple-icons/assets/12817388/6c896bc0-1995-4105-877f-5c69077ab9bb)

Where did you see the current one @adamrusted?

Reply:
I created it myself based on the colour logo - the treatment you've shown above does _not_ display well at the smaller sizes our project are generally used for - so I tried to counter that by just having the inlay.